django/test/runner.py

```diff
@@ -96,9 +94,7 @@ def printErrorList(self, flavour, errors):
             self.stream.writeln(self.separator2)
             self.stream.writeln(err)
             self.stream.writeln(self.separator2)
-            self.stream.writeln(
-                sqlparse.format(sql_debug, reindent=True, keyword_case="upper")
-            )
+            self.stream.writeln(sql_debug)
```

Comment:
This is not a fix, you simply revert 3283120cca5d5eba5c3619612d0de5ad49dcf054.

Reply:
Should I also modify the 4.2 releases note when I revert?

Reply:
Thank you, I reverted to the previous commit.

Reply:
My apologies for any confusion caused by my oversight. Based on my understanding, here are the points that require task:

1. Ensuring that SQL reserved words are not capitalized.
2. Adding a new line at the end of each query to separate it from the execution time.

Could you kindly confirm if there's anything else I might have missed? Your input is always valued and appreciated!